app/src/ui/lib/install-cli.ts

```diff
@@ -0,0 +1,64 @@
+import * as Fs from 'fs'
+import * as Path from 'path'
+
+const runas: (
+  command: string,
+  args: ReadonlyArray<string>,
+  options?: { admin: boolean }
+) => number = require('runas')
+
+/** The path for the installed command line tool. */
+export const InstalledCLIPath = '/usr/local/bin/github'
+
+/** The path to the packaged CLI. */
+const PackagedPath = Path.resolve(__dirname, 'static', 'github.sh')
```

Comment:
I'm not entirely sure I understand what you're asking, so I'll just ramble 😛

The packaged path will be the path to the script in the app bundle. The app bundle can be put anywhere. The user can move the app bundle at anytime. That would break the symlink and they'd need to reinstall it. This was the same with Classic Mac.